.devcontainer/dotnet/devcontainer.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "DotNet",
+  "image": "mcr.microsoft.com/devcontainers/javascript-node:20",
+  "forwardPorts": [8888],
+  "portsAttributes": {
+    "8888": {
+      "label": "DotNet",
+      "onAutoForward": "openBrowserOnce"
+    }
+  },
+  "secrets": {
+    "PAYPAL_CLIENT_ID": {
+      "description": "Sandbox client ID of the application.",
+      "documentationUrl": "https://developer.paypal.com/dashboard/applications/sandbox"
+    },
+    "PAYPAL_CLIENT_SECRET": {
+      "description": "Sandbox secret of the application.",
+      "documentationUrl": "https://developer.paypal.com/dashboard/applications/sandbox"
+    }
+  },
+  "containerEnv": {
+    "BASE_URL": "https://api-m.sandbox.paypal.com"
+  },
+  "build": {
+    "dockerfile": "../Dockerfile",
+    "context": "../.."
+  },
```

Comment:
Why do you need custom docker image here?  This devcontainer is already using pre-built versions of node